### PR TITLE
feat(project): add doc of project and update home url

### PR DESCRIPTION
**Pull request by Zhiw**

### CODE REVIEW

**Review by zibuyu1995 — APPROVED** [review]

File: en_US/deployments/project.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Project Management
+
+Project management is used to group deployments under different projects, making it easy to manage deployments according to projects.
```
Comment (chenx-work) [inline en_US/deployments/project.md:3]: according to the projects

File: en_US/deployments/project.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Click on the menu in the upper right corner of the deployment and select **Move Project**
+
+![move_project_menu](./_assets/move_project_menu.png)
+
+Select the target project in the project list
+
+![move_project](./_assets/move_project.png)
+
+Once you click Confirm, you can move the deployment to a new project
```
Comment (chenx-work) [inline en_US/deployments/project.md:33]: **Confirm**

File: en_US/deployments/project.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+![new_project_deployments](./_assets/new_project_deployments.png)
+
+## Update project
+
+Hover over the project to see the update button
+
+![update_project_menu](./_assets/update_project_menu.png)
+
+Click to update the project's name and description
```
Comment (chenx-work) [inline en_US/deployments/project.md:43]: Click on the button to update.....

File: en_US/deployments/project.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+![update_project](./_assets/update_project.png)
+
+## Delete project
+
+> Make sure there are no deployments under the project before deleting it
+
+Hover over the project and click the Delete button to delete the project
```
Comment (chenx-work) [inline en_US/deployments/project.md:51]: **Delete**

**Review by wuzufeng — APPROVED** [review]

**Comment by Zhiw** [comment]
@chenx-work 已修改

**Review by Adek06 — APPROVED** [review]

**Pull request merged**